recipe/build.sh

```diff
@@ -5,9 +5,10 @@ export ARCTICDB_USING_CONDA=1
 # around binary operators' template specialization expansion and compilation.
 # See: https://github.com/man-group/ArcticDB/blob/master/cpp/arcticdb/processing/operation_dispatch_binary.hpp#L451-L473
 #
+# It also has been observed to hang on `version/symbol_list.cpp`.
 # The feedstock's machine have 6.759GiB of RAM and 2 (physical?) cores so we cap it to that.
-# We build with only 2 threads to prevent these freezes from happening.
-export CMAKE_BUILD_PARALLEL_LEVEL=2
+# We build with only 1 thread to prevent these freezes from happening.
+export CMAKE_BUILD_PARALLEL_LEVEL=1
```

Comment:
This is necessary for the compilation to pass now.

Reply:
Are you sure that this is causing the build to fail? We have been using multiple threads in ArcticDB CI.